### DISCUSSION TITLE
Bring an Astro Integration to Storybook to Supercharge Development

### DISCUSSION THREAD

**Discussion by lukemcd-huge** [withastro/roadmap#391] [Proposal]
[Storybook](https://storybook.js.org) is a widely adopted tool for testing UIs and viewing components in isolation. The Storybook team has created integrated support for:

- React
- Vue
- Angular
- Web Components

Communities have added support for:

- Ember
- Svelete
- Preact
- HTML

If work could be done to add support for `.astro` files it would bring together two of the most powerful and talked about tools for frontend development. Not sure where this work would begin but wanted to see if others feel the same way.

**Comment by spjpgrd** [comment]
I'm here just to say — hell ye, that'd be rad.

**Reply by spjpgrd** [reply]
I'm still here to say, hell ye — that'd be rad.

**Reply by nour-abdulnasser** [reply]
Right there with ya

**Reply by fabirez** [reply]
Let's go! I'm here too, it would be amazing
